src/main/java/com/nutomic/syncthingandroid/syncthing/PollWebGuiAvailableTask.java

```diff
@@ -28,13 +30,13 @@
     private static final long WEB_GUI_POLL_INTERVAL = 100;
 
     /**
-     * @param url The URL of the web GUI (eg 127.0.0.1:8384).
+     * @param params The HTTPS-Cert.pem location followed by the URL of the web GUI (eg 127.0.0.1:8384).
      */
     @Override
-    protected Void doInBackground(String... url) {
+    protected Void doInBackground(String... params) {
```

Comment:
Here you're passing the cert via param, but in `GetTask`/`PostTask` you use the c'tor. Would be nice to do it consistently, maybe even with an abstract superclass.